packages/legend-query/src/stores/QueryBuilderProjectionState.ts

```diff
@@ -181,6 +182,12 @@ export class QueryBuilderSimpleProjectionColumnState extends QueryBuilderProject
     );
   }
 
+  get propertyExpression(): QueryBuilderPropertyExpressionState {
```

Comment:
As said, this is the kind of thing we try to avoid. When you add something like this, especially when you don't have any documentation nor some intense alerting name like `DONTUSEME__getPropertyExpression`, you are creating yet another thing people can accidentally use without really understanding about it.